src/main/java/core/basesyntax/strategy/handler/HandlerOperations.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.handler;
+
+public enum HandlerOperations {
+    /*
+    BALANCE ("b"),
+    PURCHASE ("p"),
+    SUPPLY ("s"),
+    RETURN ("r");
+
+    String operation;
+
+    HandlerOperations() {
+    }
+
+    HandlerOperations(String operation) {
+        this.operation = operation;
+    }
+     */
+}
```

Comment:
do we need this enum?

Reply:
I wonna use it, but cant do it. And i wonna ask it on qna.